README.md

```diff
@@ -57,6 +57,20 @@ cyclonedx-bom -o bom.xml
 cyclonedx-bom -o bom.json
 ```
 
+## Usage with docker
+
+Run `cyclonedx/cyclonedx-node` docker image inside your project folder using:
+
+```bash
+docker run --rm \
+  -v `pwd`:/src \
```

Comment:
add `""` around `pwd` - to have spaces in paths escaped
better go with `"$PWD"` ? it is universal, and unlike `pwd` it works in shells that are not bash.

Reply:
Yes, I think you are right. I will fix it.